### PR TITLE
Recipe test tweaks

### PR DESCRIPTION
**Pull request by blongden73**
This is some cosmetic changes to the recipe template based on user feedback.

- Increases the line height for the recipe. 
- Removes the read more button 
- keeps the article fully visible
- changes the width of the image so that it slightly less image and more text

**Before**
screen shot 2017-04-13 at 14 51 31

**After**
screen shot 2017-04-13 at 14 48 23

### CODE REVIEW

File: static/src/stylesheets/module/content/_recipe-article.scss [hunk 1]
```diff
@@ -310,14 +315,25 @@
     margin-bottom: $gs-baseline * 2;
     color: $neutral-1;
     max-width: 550px;
+    @include mq($from: desktop) {
```
Comment (NataliaLKB) [inline static/src/stylesheets/module/content/_recipe-article.scss]: Nit: Add one space above?

File: static/src/stylesheets/module/content/_recipe-article.scss [hunk 2]
```diff
@@ -381,19 +408,55 @@
...
     padding-top: $gs-baseline / 2;
     border-top: 1px solid $neutral-6;
+}
+
+.recipe__content {
+    padding-top: $gs-baseline / 2;
+    border-top: 1px solid $neutral-6;
 
-    p {
+    p, ul {
```
Comment (NataliaLKB) [inline static/src/stylesheets/module/content/_recipe-article.scss]: `ul` on another line

File: static/src/stylesheets/module/content/_recipe-article.scss [hunk 3]
```diff
@@ -381,19 +408,55 @@
...
+        font-size: 18px;
     }
 }
 
 .recipe__standfirst {
+    @include fs-headline(2);
     margin-bottom: $gs-gutter;
+
+    p {
+        font-size: 18px;
```
Comment (NataliaLKB) [inline static/src/stylesheets/module/content/_recipe-article.scss]: Does this not inherit from the `recipe__standfirst` ?

**Comment by paperboyo** [comment]
There is a strange drop cap-like afterthought at the beginning of the body text visible above.

**Comment by blongden73** [comment]
@paperboyo That is in the article in composer. Just a space after the "t".

File: static/src/stylesheets/module/content/_recipe-article.scss (hunk 3 above)
Comment (blongden73) [inline static/src/stylesheets/module/content/_recipe-article.scss]: yes it does :-) removed.

File: static/src/stylesheets/module/content/_recipe-article.scss (hunk 2 above)
Comment (blongden73) [inline static/src/stylesheets/module/content/_recipe-article.scss]: done.

File: static/src/stylesheets/module/content/_recipe-article.scss (hunk 1 above)
Comment (blongden73) [inline static/src/stylesheets/module/content/_recipe-article.scss]: done.

**Review by NataliaLKB — APPROVED** [review]

**Pull request merged**